userguide/tutorials/swagger_documentation.adoc

```diff
@@ -0,0 +1,111 @@
+
+. Enter values for parameters as follows:
+
+* `accountId` - The account whose payment methods are to be retrieved (example - `a21f1ca3-53ec-456b-8039-7170350c9c12`)
+
+* `withPluginInfo` - Optional, defaults to false, indicating that plugin info should not be retrieved
+
+* `includedDeleted` - Optional, defaults to false, indicating that deleted payment methods should not be included
+
+* `pluginProperty` - Optional, can be used to specify plugin properties
+
+* `audit` - Optional, defaults to NONE, indicating that audit information should not be returned
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/swagger/retrieve_account_payment_method_execution2.png[align=center]
+. Click *Execute*. This displays the following screen:
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/swagger/retrieve_account_payment_method_output.png[align=center]
+
+So, all the payment methods associated with the specified `accountId` are returned.
+
+== Execute a POST API method
+
+This section explains how you can execute a POST API method. 
+
+Suppose you would like to execute the `/1.0/kb/accounts/{accountId}/payments` method to trigger a payment. 
+You need to do the following:
+
+. Click on the `/1.0/kb/accounts/{accountId}/payments` method. This expands the method as follows:
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/swagger/trigger_payment.png[align=center]
+
+. Click on *Try it Out*. This displays the following screen:
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/swagger/trigger_payment_execution.png[align=center]
+
+. Enter values for parameters as follows:
+
+* `accountId` - The account to be charged (example - `a21f1ca3-53ec-456b-8039-7170350c9c12`)
+
+* `body` - The body of the POST request. For example, to trigger a purchase transaction, you may enter the following:
+[source,json]
+{
+  "transactionType": "PURCHASE",
+  "amount": 10,
+  "currency": "GBP"
+}
+
+* `paymentMethodId` - Optional, can be left blank. If left blank, the user's default payment method gets charged
+
+* `X-Killbill-CreatedBy` - The person/API that creates this request. You can enter any value (example - `swagger-demo`)
+
+* `X-Killbill-Reason` - The reason for creating the request. Optional, can be left blank.
+
+* `X-Killbill-Comment` - Additional comment. Optional, can be left blank.
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/swagger/trigger_payment_execution2.png[align=center]
+
+. Click *Execute*. This displays the following screen:
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/swagger/trigger_payment_execution_output.png[align=center]
+
+So, you can see that a purchase payment is triggered against the default payment method for the specified `accountId`
```

Comment:
Trailing dot

Reply:
Don't seem to find the trailing dot :), can you please let me know the line number?

Reply:
Sorry -- _missing_ trailing dot :-)